source/guide_directus.rst

```diff
@@ -0,0 +1,154 @@
+.. tag_list::
+
+Directus_ is a real-time API and App dashboard for managing SQL database content.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`the shell <basics-shell>`
+  * :manual:`Node.js <lang-nodejs>`
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`domains <web-domains>`
+
+License
+=======
+
+All relevant legal information can be found here in the GitHub_ repository of the project:
+
+  * https://github.com/directus/directus/blob/main/license
+
+Prerequisites
+=============
+
+n/a
+
+Installation
+============
+
+Create directory ``directus``.
+
+::
+
+ [isabell@stardust ~]$ mkdir -p ~/directus/logs && 
+ [isabell@stardust ~]$ cd ~/directus
+ [isabell@stardust ~]$
+
+.. note::
+
+  As CentOS 7 (uberspace) only provides ``glibc 2.17``, the ``argon2`` module on needs to be manually recompiled. This will avoid the ``Error: /lib64/libc.so.6: version GLIBC_2.25``.
+    
+  .. code-block:: bash
+    
+    [isabell@stardust ~]$ cd ~/directus
+    [isabell@stardust ~]$ npm install @mapbox/node-pre-gyp
+    [isabell@stardust ~]$ npm install argon2 --ignore-scripts
+    [isabell@stardust ~]$ npx node-pre-gyp rebuild -C ./node_modules/argon2
+    [isabell@stardust ~]$
+
+
+Initialize a new instance of Directus_
+
+::
+
+ [isabell@stardust ~]$ npx directus init
+ ? Choose your database client
+   PostgreSQL / Redshift
+   MySQL / MariaDB / Aurora
+ ❯ SQLite
+   Microsoft SQL Server
+   Oracle Database (Alpha)
+ ⠹ Installing Database Driver...
+ ? Choose your database client SQLite
+ ? Database File Path: (/home/isabell/directus/data.db)
+ Create your first admin user:
+ ? Email admin@example.com
+ ? Password **********
+ Your project has been created at /home/isabell/directus.
+ The configuration can be found in /home/isabell/directus/.env
+ [isabell@stardust ghost]$
+
+Configuration
+=============
+
+Setup daemon
+------------
+
+Create ``~/etc/services.d/directus.ini`` with the following content:
+
+.. code-block:: ini
+
+  [program:directus]
+  process_name=%(program_name)s
+  directory=%(ENV_HOME)s/directus
+  command=%(ENV_HOME)s/bin/npx directus start
+  autostart=true
+  autorestart=true
+  environment=NODE_ENV=production
+  startsecs=60
+  #stderr_logfile=%(ENV_HOME)s/logs/%(program_name)s.err.log
+  #stdout_logfile=%(ENV_HOME)s/logs/%(program_name)s.out.log
+
+.. include:: includes/supervisord.rst
+
+If it's not in state RUNNING, check your configuration.
+
+Finishing installation
+======================
+
+Point your browser to URL and log into your admin user account.
+
+Best practices
+==============
+
+Web backend
+-----------
+
+.. warning:: This exposes Directus_ to the global internet and allows anyone to query your directus database. Not recommended without authentication!
+
+.. note::
+
+    Directus_ is running on port 8055. The ``--remove-prefix`` parameter is needed if you want to run Directus_ under a sub URI like ``/directus`` instead of the root URI ``/``.
```

Comment:
Please make it as simple as possible. [Don't mention sub domains or folders](https://github.com/Uberspace/lab/blob/main/STYLE.md):
> Don't mention additional document roots. Keep it simple. Don't use subfolders. Always use the standard document root ~/html. Always assume the document root is empty.

Reply:
removed section